satysfi-fonts-dejavu.opam

```diff
@@ -20,7 +20,7 @@ extra-source "dejavu-fonts-ttf-2.37.zip" {
   ]
 }
 depends: [
-  "satysfi" {>= "0.0.3" & < "0.0.4"}
+  "satysfi" {>= "0.0.4" & < "0.0.5"}
```

Comment:
SATySFi 0.0.3 does not support src as it been introduced since 2019-02-11. If you want to include development snapshots, you can specify the version as {>= "0.0.3+dev2019.02.12" & < "0.0.5"}, otherwise {>= "0.0.4" & < "0.0.5"}